cmd/opampsupervisor/e2e_test.go

```diff
@@ -1457,3 +1458,38 @@ func findRandomPort() (int, error) {
 
 	return port, nil
 }
+
+func getOSDescription(t *testing.T) string {
```

Comment:
Not sure if there's a better way to get this test passing other than just copying this code from the opamp extension test?